scanapp/models.py

```diff
@@ -102,6 +102,87 @@ def __str__(self):
 
     scan = models.ForeignKey(Scan)
     path = models.CharField(max_length=400, help_text='Path of file scanned')
+    type = models.CharField(max_length=400, help_text='Type of the entity being scanned')
```

Comment:
At the moment, there is no proper documentation of what the fields are in ScanCode, so we may use this help text as a definition for the fields. Thanks for getting it started for us!

That said, there can be some improvements made to the text to make it more accurate. For example, in the case of type, it should just say "Type of resource: file or directory". Do not use "entity" as anything we scan is either a file or a directory. Also remove "being scanned" from the text, it is not necessary. 

Some other suggestions:

name: "Name of the file or directory"
date: "Creation date of file or directory"

etc.

Reply:
OK, that seems fair...